.evergreen/auth_aws/aws_tester.py

```diff
@@ -101,8 +101,13 @@ def setup_ecs():
         text = fid.read()
     if 'jammy' in text:
         task_definition = CONFIG[get_key('iam_auth_ecs_task_definition_jammy')]
+        if task_definition is None:
+            raise ValueError('Please set "iam_auth_ecs_task_definition_jammy" variable')
     elif 'focal' in text:
         task_definition = CONFIG[get_key('iam_auth_ecs_task_definition_focal')]
```

Comment:
This will still fail as getting a value from a dictionary that does not exist will throw a `KeyError`:

https://spruce.mongodb.com/task/mongo_node_driver_next_ubuntu2004_test_mongodb_aws_aws_4.4_auth_test_run_aws_ECS_auth_test_no_peer_dependencies_patch_fd58eec441d99493aff5c62bcc170c469efa679b_654a441561837d5366a0a964_23_11_07_14_05_26/logs?execution=1

Reply:
So I think the option check needs to happen for `get_key` and then fall back no?

Reply:
Ah, you are correct, I updated the dict access.